Gemfile.lock

```diff
@@ -106,7 +106,7 @@ GEM
       rake (>= 12.2)
       thor (~> 1.0)
     rake (13.0.6)
-    redcarpet (3.5.1)
+    redcarpet (3.6.0)
```

Comment:
I noticed a warning coming through so ended up also bumping this gem for latest patch.